go.mod

```diff
@@ -37,12 +37,17 @@ require (
 	github.com/xlab/closer v0.0.0-20190328110542-03326addb7c2 // indirect
 	github.com/xlab/suplog v1.3.1
 	go.etcd.io/bbolt v1.3.6 // indirect
-	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
+	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5 // indirect
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	google.golang.org/grpc v1.41.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
+	github.com/golang/mock v1.6.0
+	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
+)
```

Comment:
I would put these in the require above and run `go mod tidy -compat=1.17`

Reply:
OK, ran `go mod tidy -compat=1.17 `